templates/package.json.tmpl

```diff
@@ -0,0 +1,89 @@
+{
+	"name": "@liskhq/lisk-template",
+	"version": "0.1.0",
+	"description": "Library according to the Lisk protocol",
+	"author": "Lisk Foundation <admin@lisk.io>, lightcurve GmbH <admin@lightcurve.io>",
+	"license": "GPL-3.0",
+	"keywords": [
+		"lisk",
+		"blockchain"
+	],
+	"homepage": "https://github.com/LiskHQ/lisk-elements/tree/master/packages/lisk-transactions#readme",
+	"repository": {
+		"type": "git",
+		"url": "git+https://github.com/LiskHQ/lisk-elements.git"
+	},
+	"bugs": {
+		"url": "https://github.com/LiskHQ/lisk-elements/issues"
+	},
+	"engines": {
+		"node": ">=8.10 <=10",
+		"npm": ">=5"
+	},
+	"main": "dist-node/index.js",
+	"scripts": {
+		"transpile": "tsc",
+		"transpile:browsertest": "tsc -p tsconfig.browsertest.json",
+		"browserify": "browserify ./dist-node/index.js -o ./dist-browser/index.js -s liskTransactions",
+		"browserify:browsertest": "browserify ./browsertest.build/test/*.js -o ./browsertest.build/browsertest.js -s liskTransactions",
+		"uglify": "uglifyjs -nm -o ./dist-browser/index.min.js ./dist-browser/index.js",
+		"uglify:browsertest": "uglifyjs -o ./browsertest.build/browsertest.min.js ./browsertest.build/browsertest.js",
+		"clean": "./scripts/clean.sh",
+		"format": "prettier --write \"*.{ts,js,json}\" \"{src,test}/**/*.{ts,js,json}\"",
+		"lint": "tslint --format verbose --project .",
+		"lint:fix": "npm run lint -- --fix",
+		"test": "TS_NODE_PROJECT=./test/tsconfig.json nyc mocha test/{,/**/,/**/**/}/*.ts",
+		"test:watch": "npm test -- --watch",
+		"test:watch:min": "npm run test:watch -- --reporter=min",
+		"test:node": "npm run build:check",
+		"serve:start": "http-server -p 11545 ./browsertest &",
+		"serve:stop": "kill $(lsof -t -i:11545) || true",
```

Comment:
Here we had to provide a different port numbers for each package. Having them hard coded in the template may cause a problem. 

Maybe we should get the port number the user or setting an invalid port number like `-p PORT_HERE` so after initialisation force user to fix it?